server/main.lua

```diff
@@ -109,6 +109,9 @@ end)
 RegisterNetEvent('qbx_medical:server:onPlayerLaststand', function(text)
 	if GetInvokingResource() then return end
 	local src = source
+    local player = exports.qbx_core:GetPlayer(src)
+	player.Functions.SetJobDuty(false)
+    TriggerClientEvent('QBCore:Client:SetDuty', src, player.PlayerData.job.onduty)
```

Comment:
This should be triggered by core. Not here.

Reply:
removed